Moli/Core/Components/UserThumbnails/UserThumbnailView.swift

```diff
@@ -41,7 +41,7 @@ struct UserThumbnailView: View {
                     imagePlaceholder
                 }
             } else {
-                Text(getFirst2LettersOf(userMinified.streamerName, getOnlyFirstLetter: size == .thin))
+                Text(getFirst2LettersOf(userMinified.streamerName))
```

Comment:
:bomb: Tror det blev en merge error, behöver `getOnlyFirstLetter: size == .thin` för att hantera ett UI case när man använder det minsta värdet på size (`thin`)
```suggestion
                Text(getFirst2LettersOf(userMinified.streamerName), getOnlyFirstLetter: size == .thin)
```